src/invoices/InvoiceLineDetails/InvoiceLineDetails.js

```diff
@@ -79,6 +87,26 @@ class InvoiceLineDetails extends Component {
       />
     );
 
+    const lastMenu = (
+      <PaneMenu>
+        {tagsEnabled && (
+          <FormattedMessage id="ui-invoice.showTags">
```

Comment:
I think it should be a component

Reply:
IMHO, no needed. Here I use `<IconButton />`

Reply:
Created separate component `<TagBadge />`